diff --git a/app/src/terminal/local_tty/windows/conpty_api.rs b/app/src/terminal/local_tty/windows/conpty_api.rs
--- a/app/src/terminal/local_tty/windows/conpty_api.rs
+++ b/app/src/terminal/local_tty/windows/conpty_api.rs
@@ -12,7 +12,7 @@ use winreg::enums::HKEY_LOCAL_MACHINE;
 
 const BUNDLED_CONPTY_DLL: &str = "conpty.dll";
 const SYSTEM_CONPTY_DLL: &str = "kernel32.dll";
-const BUNDLED_CONPTY_MIN_WINDOWS_BUILD: u32 = 18362;
+const BUNDLED_CONPTY_MIN_WINDOWS_BUILD: u32 = 17763;
 
 const CREATE_PSUEDOCONSOLE_FN_NAME: &str = "CreatePsuedoConsole";
 const RESIZE_PSUEDOCONSOLE_FN_NAME: &str = "ResizePsuedoConsole";
diff --git a/app/src/terminal/local_tty/windows/mod.rs b/app/src/terminal/local_tty/windows/mod.rs
--- a/app/src/terminal/local_tty/windows/mod.rs
+++ b/app/src/terminal/local_tty/windows/mod.rs
@@ -60,7 +60,7 @@ unsafe impl Sync for ShareableHandle {}
 
-pub(super) enum PtyPipe {
+pub enum PtyPipe {
     Duplex(mio::windows::NamedPipe),
     Split {
         reader: mio::windows::NamedPipe,
